backend/ops_api/tests/ops/notification/test_notification.py

```diff
@@ -147,7 +147,7 @@ def test_notifications_get_by_oidc_id(auth_client, loaded_db, notification):
 def test_notifications_get_by_is_read(auth_client, loaded_db, notification_is_read_is_true):
     response = auth_client.get("/api/v1/notifications/?is_read=False")
     assert response.status_code == 200
-    assert len(response.json) == 21
+    assert len(response.json) == 41
```

Comment:
similar to the comment above - would be better to compare the DB count to the response count and not use specific numbers here

Reply:
k, I'll make that change in the next round